common/djangoapps/third_party_auth/models.py

```diff
@@ -244,6 +261,12 @@ class OAuth2ProviderConfig(ProviderConfig):
             # To be precise, it's set by AUTHENTICATION_BACKENDS - which aws.py sets from THIRD_PARTY_AUTH_BACKENDS
         )
     )
+    provider_slug = models.SlugField(
```

Comment:
Got it. I missed the fact that `LTIProviderConfig` also extends `ProviderConfig`. Thanks for the clarification. Let's stick with what you have.